### DISCUSSION TITLE
一般

### DISCUSSION THREAD

**Discussion by naoland** [naoland/nem-kaitai-shinsho#12] [General]
質問、連絡、ディスカッションなどにご利用ください。特に制限はないですが、大人としてのふるまいをお願いします。

**Comment by shizuilab** [comment]
おはようございます。よろしくお願いします。

**Comment by naoland** [comment]
よろしくお願いします。

2章、3章については、文章を整理した方がよいかと思います。
また、Markdown記法では文字色を変更できませんので、文字色に頼らない強調の仕方を検討した方がよいかと思います。

お伝えしましたように、一通り私の方で作業が終わるまでは変更しないでください。

ブランチ名の付け方は、とりあえず、 **#イシュー番号-username** としようかと思ってます。

例）#14-naoland

**Reply by shizuilab** [reply]
初期の頃は全部やる予定ではなかったので、勝手に記事を省略して独自解説したりしてます。そのあたりは、書き直したほうが良いかどうか考えておきますね。

**Reply by naoland** [reply]
はい、「初期の頃は全部やる予定ではなかった」は重々承知です。
私が持ちかけて、面倒をおかけしてすみませんｗ。

**Reply by shizuilab** [reply]
3章については、3.1, 3.2はSymbol白書の該当部とほぼ一緒です。Symbol白書のものをそのまま流用しても良いと思います。3.3については、4.1のところに組み込まれています。目次に合わせるには、若干の修正は必要ですが、そこから抜き出せば良いと思います。

https://nemlog.nem.social/blog/8404

**Comment by naoland** [comment]
@shizuilab 
プログラムの実行環境は示しておいた方がよいかもしれません。
使用したライブラリなども。

**Reply by shizuilab** [reply]
ちょっと、今学生実習とNEMTUS負荷テストが重なっててんてこ舞いなので、先に進めておいてください。時間ができたらとりかかります。

**Reply by shizuilab** [reply]
Symbol白書の3.1で、コードを一部書き直してあります。Python2で動くはずです。

https://nemlog.nem.social/blog/38109

**Reply by naoland** [reply]
全然慌ててませんので、しっかりとお時間のある時にお願いします。
私の方も、1日にちょっとしか進められてません。

Python2はすでに正式にサポート切れですので、将来的にお時間がとれるようでしたら、Python3やその他の言語（JavaScriptなど）で書き直すのもありかもしれません。

**Comment by naoland** [comment]
そういえば、今は岐阜県にお住まいなのですか？
私は2006年ごろは大垣のソフトピアジャパン近くに住んでました。
